src/tasks/leveling.ts

```diff
@@ -104,24 +117,60 @@ const freeFightMonsters: Monster[] = $monsters`Witchess Bishop, Witchess King, W
 const craftedCBBFoods: Item[] = $items`honey bun of Boris, roasted vegetable of Jarlsberg, Pete's rich ricotta, plain calzone`;
 const craftedCBBEffects: Effect[] = craftedCBBFoods.map((it) => effectModifier(it, "effect"));
 let triedCraftingCBBFoods = false;
+
+interface StatTo<X> {
+  Muscle: X;
+  Mysticality: X;
+  Moxie: X;
+}
+
+const mainStatStr = myPrimestat().toString();
+const statEffects: StatTo<Effect[]> = {
```

Comment:
Where is this called? Did you forget to add it to `usefulEffects`?

Reply:
I did; I couldn't figure out how to concatenate the strings and ended up solving it a different way, but didn't include that here. You might hate my fix.

Reply:
You are calling `...statEffects` everywhere that `...usefulEffects` is being called. Might as well add it in here
```ts
const usefulEffects: Effect[] = [
  // Stats
  $effect`Big`,
  $effect`Feeling Excited`,
  $effect`Triple-Sized`,
  $effect`substats.enh`,
  $effect`Hulkien`,
  $effect`Broad-Spectrum Vaccine`,
  $effect`Song of Bravado`,
  ...statEffects,

  // ML
  $effect`Pride of the Puffin`,
  $effect`Aloysius' Antiphon of Aptitude`,

  // Spell dmg
  $effect`Carol of the Hells`,
];
```

Reply:
Resolved.